dts/bindings/cpu/cpu.yaml

```diff
@@ -10,3 +10,9 @@ properties:
       type: int
       required: false
       description: Clock frequency in Hz
+    pm-states:
+       type: phandles
+       required: false
+       description: |
+         List of power management states supported
+         by this cpu.
```

Comment:
I think this will probably look weird in the [bindings index](https://docs.zephyrproject.org/latest/reference/devicetree/bindings.html); did you consider putting all this text on one line?

Reply:
This particularly one I can, but what happens with big lines ? Can't we break them ?

Reply:
> This particularly one I can, but what happens with big lines ? Can't we break them ?

Yeah, for sure, you can break them, they just get formatted in a big text block if you do. Here's an example:

https://docs.zephyrproject.org/latest/reference/devicetree/bindings/ilitek%2Cili9340-spi.html#ilitek-ili9340-spi

This cpu binding looks like it will fit on one line, which may be easier to read.